_targets/build.project.armv7m7-imxrt106x

```diff
@@ -48,24 +48,23 @@ KERNEL_OFFS=$((0x9000))
 export PROGS
 
 
-
 #
 # Project specific build
 #
 
 
 b_mkscript() {
-	ksz=$((($(stat -c "%s" ${PREFIX_PROG_STRIPPED}phoenix-${TARGET}.elf) + $SIZE_PAGE - 1)&$PAGE_MASK))
-	poffs=$(($KERNEL_OFFS + $ksz))
+	ksz=$((($(stat -c "%s" "${PREFIX_PROG_STRIPPED}phoenix-${TARGET}.elf") + SIZE_PAGE - 1) & PAGE_MASK))
+	poffs=$((KERNEL_OFFS + ksz))
 
 	for prog in "${PROGS[@]}"; do
-		psz=$((($(stat -c "%s" ${PREFIX_PROG_STRIPPED}$prog) + $SIZE_PAGE - 1)&$PAGE_MASK))
-		printf "@%s(%x:%x)\n" $prog $poffs $psz >> $PREFIX_BUILD/plo/script.plo
-		((poffs+=$psz))
+		psz=$((($(stat -c "%s" "${PREFIX_PROG_STRIPPED}$prog") + SIZE_PAGE - 1) & PAGE_MASK))
+		printf "@%s(%x:%x)\n" "$prog" "$poffs" "$psz" >> "$PREFIX_BUILD/plo/script.plo"
+		((poffs+=psz))
 	done
 
-	cat _targets/armv7m7-imxrt106x.plo >> $PREFIX_BUILD/plo/script.plo
-	printf "\0" >> $PREFIX_BUILD/plo/script.plo
+	cat _targets/armv7m7-imxrt106x.plo >> "$PREFIX_BUILD/plo/script.plo"
+	printf "\0" >> "$PREFIX_BUILD/plo/script.plo"
```

Comment:
just a quick note - appending the "images placement description" to the `plo` can't be the final solution as we won't be able to change them during FW upgrade (without overwriting PLO - and this is something you **never** want to do in the field). I'm assuming the current solution would evolve in that direction in the future.